src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,85 @@
+package duke;
+
+import exception.DukeException;
+import java.util.ArrayList;
+import task.Task;
+
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /** @return int number of tasks in tasklist */
+    public int size() {
+        return this.tasks.size();
+    }
+
+    /** @return Boolean whether task list is empty */
+    public Boolean isEmpty() {
+        return this.tasks.isEmpty();
+    }
+
+    /** @return ArrayList<String> returns an array of tasks in string format */
+    public ArrayList<String> getAllTaskString() {
+        ArrayList<String> taskString = new ArrayList<>();
```

Comment:
Method can have a better name e.g. `getAllTasksAsStrings()`.
The arraylist should have a plural name as well. E.g. `tasksAsStrings`.